### PR TITLE
Add brief intro text to "About" page.

### PR DESCRIPTION
**Pull request by csadorf**
And fix the list notation.

### CODE REVIEW

**Comment by sphuber** [comment]
@csadorf does this address #4024 ? If so, should this be linked from the home page? I think currently to see this people will have to click in the sidebar on the "What is AiiDA". This can be OK, but I thought we were discussing at some point to have this on the landing page. Or do we just want to keep the panels there now as they are right now?

**Comment by csadorf** [comment]
@sphuber Yes, it's supposed to address #4024 . You are correct that there is currently no link to that page. I think I'd prefer to keep the landing page clutter free and that is quite a lot of text. My suggestion during the meeting was to just rely on aiida.net for the motivation and feature overview in which case we could remove this page altogether. Alternatively we could link to it from the landing page, .e.g., change the target of the link behind the first word ("AiiDA").

**Comment by chrisjsewell** [comment]
> should this be linked from the home page? 

Yeh maybe ideally it would be. I just couldn't find an obvious place for it on my beautifully designed home page lol

**Comment by csadorf** [comment]
@chrisjsewell What do you think about changing the target of the "AiiDA" link?

**Comment by sphuber** [comment]
> @chrisjsewell What do you think about changing the target of the "AiiDA" link?

That would be a nice clean solution, but might get unnoticed. How about adding a single phrase after  
> AiiDA is an open-source framework implemented in Python that aims to help researchers with managing simple to complex workflows and making them fully reproducible.

to say something like:

> Have a look at :ref:`What is AiiDA<link>` to see an overview of the features .

**Comment by chrisjsewell** [comment]
> How about adding a single phrase

Yeh go for it 👍

File: docs/source/intro/about.rst [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
--  **Open source:** AiiDA is released under the `MIT open source license <LICENSE.txt>`__
+*  **HPC interface:** Move your calculations to a different computer by changing one line of code. AiiDA is compatible with schedulers like `SLURM <https://slurm.schedmd.com>`__, `PBS Pro <https://www.pbspro.org/>`__, `torque <http://www.adaptivecomputing.com/products/torque/>`__, `SGE <http://gridscheduler.sourceforge.net/>`__ or `LSF <https://www.ibm.com/support/knowledgecenter/SSETD4/product_welcome_platform_lsf.html>`__ out of the box.
+*  **Plugin interface:** Extend AiiDA with `plugins <https://aiidateam.github.io/aiida-registry/>`__ for new simulation codes (input generation & parsing), data types, schedulers, transport modes and more.
+*  **Open Science:** Export subsets of your provenance graph and share them with peers or make them available online for everyone on the `Materials Cloud <https://www.materialscloud.org>`__.
+*  **Open source:** AiiDA is released under the `MIT open source license <LICENSE.txt>`__
 
 See also the `AiiDA home page <http://www.aiida.net/>`__.
```
Comment (chrisjsewell) [inline docs/source/intro/about.rst:18]:
```suggestion
See also the `AiiDA testimonials <http://www.aiida.net/testimonials/>`__ and `AiiDA publications http://www.aiida.net/science/>`__ pages.
```

**Comment by chrisjsewell** [comment]
BTW, the "quick_install" link on http://www.aiida.net/download/ will need changing when this is merged

**Comment by chrisjsewell** [comment]
LGTM 👍

**Comment by sphuber** [comment]
> BTW, the "quick_install" link on http://www.aiida.net/download/ will need changing when this is merged

I can take care of this, but only once we have merged `docs-revamp`. @csadorf can you add an item to your to-do list for the merge so you can ping me to update the website? Thanks

**Comment by csadorf** [comment]
> > BTW, the "quick_install" link on http://www.aiida.net/download/ will need changing when this is merged
> 
> I can take care of this, but only once we have merged `docs-revamp`. @csadorf can you add an item to your to-do list for the merge so you can ping me to update the website? Thanks

Thx! Already did. :)

**Pull request merged**